cmd/cluster/aws/create.go

```diff
@@ -59,8 +57,9 @@ type ValidatedCreateOptions struct {
 
 func (o *RawCreateOptions) Validate(ctx context.Context, opts *core.CreateOptions) (core.PlatformCompleter, error) {
 	// Perform multi-arch validations except for render
+	var err error
 	if !opts.Render {
-		err := validateMultiArch(ctx, o, opts)
+		o.MultiArch, err = core.ValidateMultiArch(ctx, o.MultiArch, opts)
```

Comment:
why is this here and not in the core Validate?

Reply:
This has been removed and now only core Validate does the check.